docs/source/events/index.rst

```diff
@@ -54,6 +54,7 @@ All events are encoded as JSON. On top of the JSON built-in types the events can
 - date: ISO 8601 encoded dates either with an explicit offset or in UTC
 - interval: ISO 8601 time interval
 - money: Money value encoded as string of the floating point value and the ISO 4217 code separated by a space
```

Comment:
Give an example of what this might look like

Reply:
I’ve moved this after the float, and added examples.